src/main/java/com/echobox/api/linkedin/types/organization/Organization.java

```diff
@@ -0,0 +1,296 @@
+   * The industries associated with the entity.
+   */
+  @Getter
+  @Setter
+  private List<URN> industries;
+  
+  /**
+   * The locale-specific description of the entity.
+   */
+  @Getter
+  @Setter
+  @LinkedIn
+  private String localizedDescription;
+  
+  /**
+   * The locale-specific name of the entity.
+   */
+  @Getter
+  @Setter
+  @LinkedIn
+  private String localizedName;
+  
+  /**
+   * The locale-specific, admin-defined specialty tags of the entity.
+   */
+  @Getter
+  @Setter
+  @LinkedIn
+  private List<String> localizedSpecialties;
+
+  /**
+   * The locale-specific website of the entity.
+   */
+  @Getter
+  @Setter
+  @LinkedIn
+  private String localizedWebsite;
+  
+  /**
+   * List of locations for the entity.
+   */
+  @Getter
+  @Setter
+  @LinkedIn
+  private List<LocationInfo> locations;
+  
+  /**
+   * Entity's name.
+   */
+  @Getter
+  @Setter
+  @LinkedIn
+  private MultiLocaleString name;
+
+  /**
+   * The parent organization.
+   */
+  @Getter
+  @Setter
+  @LinkedIn
+  private ParentRelationship parentRelationship;
+
+  /**
+   * Admin-defined specialty
+   */
+  @Getter
+  @Setter
+  @LinkedIn
+  private List<Specialty> specialties;
+
+  /**
+   * Entity's unique name used in URLs.
+   */
+  @Getter
+  @Setter
+  @LinkedIn
+  private String vanityName;
+
+  /**
+   * Tag indicating version.
+   */
+  @Getter
+  @Setter
+  @LinkedIn
+  private String versionTag;
+
+  /**
+   * Entity's website.
+   */
+  @Getter
+  @Setter
+  @LinkedIn
+  private MultiLocaleString website;
+
+  /**
+   * Range of the number of staff associated with this entity.
+   */
+  @Getter
+  @Setter
+  @LinkedIn
+  private String staffCountRange;
+
+  /**
+   * School-specific attributes of the organization. 
+   * If this field exists, then this entity is a school.
+   */
+  @Getter
+  @Setter
+  @LinkedIn
+  private SchoolAttribute schoolAttributes;
+
+  /**
+   * The image used in the Overview tab on the organization's page.
+   */
+  @Getter
+  @Setter
+  @LinkedIn
+  @Deprecated
+  private CroppedImage overviewPhoto;
+
+  /**
+   * The image used in the Overview tab on the organization's page.
+   */
+  @Getter
+  @Setter
+  @LinkedIn
+  private CroppedImage overviewPhotov2;
+  
+  /**
+   * The entity’s logo.
+   */
+  @Getter
+  @Setter
+  @LinkedIn
+  @Deprecated
+  private CroppedImage logo;
+
+  /**
+   * The entity’s logo. The sizes may vary greatly, 
+   * i.e., 50x50, 100x60, 400x400, so clients should handle the given height and width accordingly.
+   */
+  @Getter
+  @Setter
+  @LinkedIn
+  private CroppedImage logoV2;
+
+  /**
+   * Status of the organization, such as operating or out of business.
+   */
+  @Getter
+  @Setter
+  @LinkedIn
+  private String organizationStatus;
+
+  /**
+   * Type of organization.
+   */
+  @Getter
+  @Setter
+  @LinkedIn
+  private String organizationType;
+  
+  @JsonMappingCompleted
+  protected void jsonMappingCompleted(JsonMapper jsonMapper) {
+
+    if (industriesRaw != null) {
```

Comment:
Could this be done in the DefaultJsonMapper directly?

Reply:
Adding:

```
      if (type.equals(URN.class)) {
        return (T) new URN(json);
      }
```

to DefualtJsonMapper:267 seems to work...I seem to remember Clement having this there as well. I am not sure this is exactly a neat solution but would be nice to have the logic centralised....there is already an exception at DefualtJsonMapper:179 so maybe we could add another here?

Reply:
Yeah, ideally we don't want to change the DefaultJsonMApper too much from the restFB impl incase there are breaking changes - we'd need to make sure we run a diff